server/db/driver/pg/internal/matches.go

```diff
@@ -0,0 +1,43 @@
+package internal
+
+const (
+	CreateMatchesTable = `CREATE TABLE IF NOT EXISTS %s (
+		matchid BYTEA PRIMARY KEY,
+		takerOrder BYTEA,      -- INDEX this
+		takerAccount BYTEA,    -- INDEX this
+		makerOrder BYTEA,      -- INDEX this
+		makerAccount BYTEA,    -- INDEX this
+		quantity INT8,
+		rate INT8,
+		status INT2,
+		sigTakerMatch BYTEA,
+		sigMakerMatch BYTEA,
+		sigTakerAudit BYTEA,
+		sigMakerAudit BYTEA,
+		sigTakerRedeem BYTEA,
+		sigMakerRedeem BYTEA
```

Comment:
Hopefully #75 will reduce the 4 txids + 4 vouts to 4 just coin IDs. I'm actually thinking of just dropping the timestamp from the serialization. I can't think of a whole lot that the timestamp does for us.

Reply:
Just a couple more thoughts:

Like the signatures, we could just save the **last** coin ID from each client, instead of all of them. That would get our needs down to 2 coin IDs, 2 signatures, and 1 script. 

For fully negotiated swaps, the script, which is obviously pretty large, can be deleted, because it can be retrieved later from the blockchain.

Reply:
I don't see a reason to hold on to the timestamp either, except perhaps to check in retrospect if those coins were spent at that time or not.  (off topic: we'll have to ensure the asset backends also check mempool when verifying coins are unspent and available, if that's not done already -- can't recall).

Regarding last (signature, coin ID) vs. all of them for each step, I think we'd want the ability to point to each on-chain txn (and which outputs were relevant) that was part of a swap.